cli/tests/setup_suite.bash

```diff
@@ -394,6 +394,8 @@ home_setup() {
 # `{setup,teardown}_suite' functions must be defined in `setup_suite.bash'
 # files, AND keep in mind that `SET_TESTS_DIR' will likely differ.
 common_suite_setup() {
+  export FLOX_FEATURES_COMPOSE='true'
```

Comment:
note nonblocking: we should probably set `flox.features.compose = true` in `flox_instance()`

Reply:
I'm unsure because the unit tests feel more targetted. I'll have a think about whether I have more evidence to back up that "feel" though.